lib/jellyfish/event.ex

```diff
@@ -16,8 +20,32 @@ defmodule Jellyfish.Event do
   @pubsub Jellyfish.PubSub
   @valid_topics [:server_notification, :metrics]
 
-  def broadcast(topic, message) when topic in @valid_topics do
+  def broadcast_metrics(message) do
+    topic = :metrics
+    Phoenix.PubSub.broadcast(@pubsub, Atom.to_string(topic), {topic, message})
+  end
+
+  def broadcast_server_notification({type, _content} = message, webhook_url)
+      when type in [:room_deleted, :room_crashed] do
+    topic = :server_notification
+    Phoenix.PubSub.broadcast(@pubsub, Atom.to_string(topic), {topic, message})
+
+    content = to_proto({topic, message})
+
+    notification = %ServerMessage{content: content} |> ServerMessage.encode()
+
+    Room.send_webhook_notification(notification, webhook_url)
+  end
+
+  def broadcast_server_notification(message, room_id) do
+    topic = :server_notification
     Phoenix.PubSub.broadcast(@pubsub, Atom.to_string(topic), {topic, message})
+
+    content = to_proto({topic, message})
+
+    notification = %ServerMessage{content: content} |> ServerMessage.encode()
+
+    Room.receive_room_notification(room_id, notification)
   end
```

Comment:
Maybe it would be better to create a separate process that will be responsible for subscribing for events and sending notificaitons using webhooks. This way we wouldn't have to modify event.ex. If neeed we could put webhook url inside Registry